Lombiq.Tests.UI/Services/GitHubActionsGroupingTestOutputHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Lombiq.Tests.UI.Models;
+using System;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Services;
+
+internal sealed class GitHubActionsGroupingTestOutputHelper : ITestOutputHelper
```

Comment:
Document how this can be utilized, similar to TeamCity metadata reporting, both [in the wiki]https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2455732262/Continuous+Integration) and in the Readme.

Reply:
I'm not entirely sure what you have in mind, but I've added an entry to the wiki and mentioned them in the main and UI testing readmes. As mentioned in the other point it's environmentally activated so nothing in particular needs to be done to utilize it.